### DISCUSSION TITLE
I would like to use a system similar to ACE

### DISCUSSION THREAD

**Discussion by LCDC1** [acemod/ACE3#9118] [General]
I am making a game (which might never come to fruition) and would like to use a UI system very similar to ACE's which your work will heavily inspire, you would be credited. I never use GitHub and the best way to contact me is on discord     LCDC#5597,     make it known that you aren't a scammer and reference this post, or I will treat you like one and try to waste your time. Thank you for your time

**Comment by jonpas** [comment]
Demands to get permission - that's a first for me.

No, we won't do what you say.
